docs/source/rbac.rst

```diff
@@ -292,108 +295,197 @@ For example:
 
 .. _rbac-using_rbac:
 
+Automatically granting roles based on the user LDAP group membership
+--------------------------------------------------------------------
+
+.. note::
+
+   This functionality is only available in |st2| v2.3.0 and above when LDAP auth backend is used
+   for authentication.
+
+In addition to manually assigning roles to the users based on the definitions in the
+``/opt/stackstorm/rbac/assignments/`` directory, |st2| also supports automatically granting roles
+to users on authentication based on the LDAP groups membership.
```

Comment:
I think this line needs to be "...|st2| also supports automatically granting roles to users upon authentication, based on LDAP groups membership."